wip-tec/en/WebServicesAccess.md

```diff
@@ -27,9 +41,27 @@ When access to collaboration tools and sites is restricted, instead of increasin
 Users will find a way around the blocks if it makes their life easier.
 Security is more than just locking things down; user experience must also be considered.
 
+#### Bandwidth
+
+The R&D team has been piloting a to route traffic to site that use a lot of bandwidth directly to the Internet.
+This type of solution could mitigate the risk of making network performance worse and could even reduce the stress on the corporate network.
+It would also eliminate the need to block sites that are flagged for using too much bandwidth.
+
+#### Digital Worker Profiles
+
+If not all groups currently require or want more open access to the Internet, we can expect that more and more of them will in the future.
+Current branches that would benefit:
+
+- IITB
+- ...
+
+#### Awareness
+
 Instead of banning access to certain tools and sites, making access open by default and encouraging the secure use will result in risks being better controlled.
 Informing users of expected behaviours when using the Internet will help them to protect against potential confidentiality or privacy breaches and to comply with existing ESDC and GC codes of conduct and policies.
 
+This information is already available on the Intranet and routinely communicated to users through, popups, emails and mandatory trainings.
+
 ## Next Steps
```

Comment:
Reword. Some of that stuff is outside of our mandate (e.g. training opportunities).
Should you approve, the IT Strategy team will:

1- Work with the Operations team to start grouping ESDC employees into work profiles
2- Work with IT Security and DSO to open up access to online services based on work profiles, starting with the lowest risks ones
3- Work with IT Security and DSO to identify the strategic objectives and priority actions from the Departmental Security Plan that needs to be addressed in order to open up access to online services
4- Research, evaluate, and recommend Open Source solutions that would satisfy the risk mitigation needs outline above

Reply:
As much as I would like to do 4., isn't more the mandate of the R&D team?

Reply:
@gabrielcossette , yeah. We'll have to engage them.
Maybe reward as "Leverge the IITB Innovation Framework to research, evaluate,  and recommend...."?